### PR TITLE
(GH-994) Add ignore-pinned switch to outdated command.

### LINKED ISSUE #994: choco outdated should list pinned packages separate from non-pinned ones.
### What You Are Seeing?

When issuing "choco outdated", it lists packages that I have previously added to the pinned list.
### What is Expected?

I would prefer to see the pinned packages listed separately from the other ones that are not pinned. Or, add a command line switch to have pinned packages NOT included in the list.
### How Did You Get This To Happen? (Steps to Reproduce)

Pin a package that has an upgrade that I have not pulled down locally. Issue "choco outdated" and it lists the pinned package right along with all the other ones. This makes it hard to see the ones I may want to upgrade from the ones I do not.

### PR DESCRIPTION
**Pull request by woodburysoft**
Previously the 'choco outdated' command listed all packages regardless
of whether they were pinned or not. Add an 'ignore-pinned' switch that
skips checking available upgrades for all packages that are pinned.

Closes #994

### CODE REVIEW

File: src/chocolatey/infrastructure.app/commands/ChocolateyOutdatedCommand.cs [hunk 1]
```diff
@@ -50,7 +50,10 @@ public virtual void configure_argument_parser(OptionSet optionSet, ChocolateyCon
                      option => configuration.SourceCommand.Certificate = option.remove_surrounding_quotes())
                 .Add("cp=|certpassword=",
                      "Certificate Password - the client certificate's password to the source. Defaults to empty. Available in 0.9.10+.",
-                     option => configuration.SourceCommand.CertificatePassword = option.remove_surrounding_quotes())
+                     option => configuration.SourceCommand.CertificatePassword = option.remove_surrounding_quotes())                
+                .Add("ip|ignore-pinned",
```
Comment (ferventcoder) [inline src/chocolatey/infrastructure.app/commands/ChocolateyOutdatedCommand.cs]: Let's not have a short version of this. At least for now. Just `--ignore-pinned`.

**Comment by ferventcoder** [comment]
You may also want to update tab completion with the new field - https://github.com/chocolatey/choco/blob/7ff013f4ffc2945efcb7bddd242401a9b9d6adac/src/chocolatey.resources/helpers/ChocolateyTabExpansion.ps1#L47

**Comment by ferventcoder** [comment]
Please also change the commit from `[GH-994]` to `(GH-994)` for consistency. Thanks!

**Comment by woodburysoft** [comment]
I've now made the requested changes.

**Comment by ferventcoder** [comment]
Thanks for your contribution! Rebased onto stable, fixed up the merge conflicts and merged at https://github.com/chocolatey/choco/commit/5e4d9a65140d385bade5469d8e026675983f588c.

**Pull request closed without merging**